# this variable holds either "human" or "robot" to state who is assigned with the task
string assignedTo
---
bool success
